packages/victory-errorbar/src/error-bar.js

```diff
@@ -17,7 +17,8 @@ const renderBorder = (props, error, type) => {
     x1: vertical ? error[type] : props.x - props.borderWidth,
     x2: vertical ? error[type] : props.x + props.borderWidth,
     y1: vertical ? props.y - props.borderWidth : error[type],
-    y2: vertical ? props.y + props.borderWidth : error[type]
+    y2: vertical ? props.y + props.borderWidth : error[type],
+    "data-type": `border-${type}`
```

Comment:
I don't usually like adding arbitrary props like this, but previously we were getting this type by string matching the key, and that felt wrong to me.